test/test_other.py

```diff
@@ -8523,13 +8523,12 @@ def test_exported_runtime_methods_metadce(self, use_legacy_name):
   def test_legalize_js_ffi(self):
     # test disabling of JS FFI legalization
     for (args, js_ffi) in [
-        (['-sLEGALIZE_JS_FFI=1', '-sSIDE_MODULE', '-O1', '-sEXPORT_ALL'], True),
-        (['-sLEGALIZE_JS_FFI=0', '-sSIDE_MODULE', '-O1', '-sEXPORT_ALL'], False),
-        (['-sLEGALIZE_JS_FFI=0', '-sSIDE_MODULE', '-O0', '-sEXPORT_ALL'], False),
+        (['-sLEGALIZE_JS_FFI=1', '-sSIDE_MODULE', '-O1'], True),
+        (['-sLEGALIZE_JS_FFI=0', '-sSIDE_MODULE', '-O1'], False),
+        (['-sLEGALIZE_JS_FFI=0', '-sSIDE_MODULE', '-O0'], False),
+        (['-sLEGALIZE_JS_FFI=1', '-sWARN_ON_UNDEFINED_SYMBOLS=0', '-O0'], True),
```

Comment:
Looks like the fourth one here is new? Was this missing coverage?

Reply:
Yup I wanted to check both the positive and negative case with and without `-sSIDE_MODULE`.

Reply:
I'm not sure the `-sSIDE_MODULE` thing is actually important here.. it might have just been away to allow undefined imports, but since I'm not sure I'll decided to leave it in for max coverage.

Reply:
Makes sense.